runtime/cudaq/algorithms/optimizer.h

```diff
@@ -70,9 +70,9 @@ class optimizable_function {
 /// to third party libraries. This interface provides an optimize(...) method
 /// that takes the number of objective function input parameters
 /// (the dimension), and a user-specified objective function that takes the
-/// function input parameters as a immutable (const) vector<double> reference
+/// function input parameters as a immutable (`const`) vector<double> reference
 /// and a mutable vector<double> reference modeling the current iteration
-/// gradient vector (df / dx_i, for all i parameters). This function
+/// gradient vector (dF / dx_i, for all i parameters). This function
```

Comment:
nit: Could use the `\f(` and `\f)` formula tags here?

Reply:
As you noted in https://github.com/NVIDIA/cuda-quantum/issues/1073 (after this comment), the LaTeX formula tags are not currently available.